### PR TITLE
Fix double "vector" menus in French

### PR DESCRIPTION
**Pull request by DelazJ**
due to different translations of vector menu

### CODE REVIEW

**Comment by Gustry** [comment]
Great, thanks a lot !
Can it be backported to 2.14 too ?

**Comment by DelazJ** [comment]
@Gustry Do you think it's necessary? Afaict, the double vector menus issue appeared with the replacement of fTools functions by Processing algorithm (in 2.16)

**Comment by Gustry** [comment]
Hum, mea culpa, time flies !

File: i18n/qgis_fr.ts [hunk 1]
```diff
@@ -13172,12 +13172,12 @@ Veuillez le configurer avant d&apos;exécuter les algorithmes LAStools.</transla
     <message>
         <location filename="../src/ui/qgisapp.ui" line="897"/>
         <source>Rotate Point Symbols</source>
-        <translation>Rotation des symboles de points</translation>
+        <translation>Pivoter des symboles de points</translation>
```
Comment (sbrunner) [inline i18n/qgis_fr.ts]: « Pivoter les symboles ponctuelles » looks relay better for me...

File: i18n/qgis_fr.ts [hunk 2]
```diff
@@ -13172,12 +13172,12 @@ Veuillez le configurer avant d&apos;exécuter les algorithmes LAStools.</transla
...
     </message>
     <message>
         <location filename="../src/ui/qgisapp.ui" line="909"/>
         <source>Offset Point Symbol</source>
-        <translation>Symbole ponctuel décalé</translation>
+        <translation>Décaler des symboles de points</translation>
```
Comment (sbrunner) [inline i18n/qgis_fr.ts]: « Décaler des symboles ponctuelles » looks relay better for me...

Comment (rouault) [inline i18n/qgis_fr.ts]: ponctuels sans 'le' brûlera moins les yeux ;-)

Comment (sbrunner) [inline i18n/qgis_fr.ts]: Corrigé merci :-)

File: i18n/qgis_fr.ts (hunk 1 above)
Comment (yjacolin) [inline i18n/qgis_fr.ts]: "Pivoter les symboles ponctuels" +1

File: i18n/qgis_fr.ts (hunk 2 above)
Comment (DelazJ) [inline i18n/qgis_fr.ts]: mais avec le 's' :-), à moins que ton inconscient proposait de tout mettre au singulier, ce à quoi je souscris (pour ce point tout comme le commentaire du dessus, malgré le plural utilisé). On a un outil `Pivoter l'entité` donc ce serait cohérent d'avoir un `Pivoter le symbole ponctuel` et un `Décaler le symbole ponctuel`.
Si ça convient, let me know et je corrige.

Comment (sbrunner) [inline i18n/qgis_fr.ts]: +1 :-)

Comment (jef-n) [inline i18n/qgis_fr.ts]: Please don't use encrypted comments. ;)

**Comment by Gustry** [comment]
Hi, is this patch working for you?
I was using QGIS in english these last few months, but I'm back in French these last few days and I still have the double "vector" menus : 

screen shot 2017-08-14 at 14 09 25

And QGIS is hiding me some icons in my Mac toolbar.

**Comment by DelazJ** [comment]
@Gustry In the Processing Options, hit the "Reset to defaults" button. Should fix that (I can't tell why it's needed though).

**Comment by Gustry** [comment]
Merci ! It works

**Pull request merged**